### PR TITLE
Fixed Get Original Email bugs

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
In Progress

## Description
Get Original Email now searches for the original email in the mailbox of the user who *forwarded* the mail (the recipient of the phishing mail), and not in the mailbox that is being listened to by the phishing investigation playbook.
Also, the original email is now extracted (or not extracted) depending on a flag that the user can set as playbook input.

## Related Issues
fixes: https://github.com/demisto/etc/issues/14154

## Required version of Demisto
4.0

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Playbooks/playbook-Get_Original_Email_-_Generic.yml [hunk 1]
```diff
@@ -76,32 +76,52 @@ tasks:
...
+    taskid: 96353c64-dc04-4475-89a8-f0f90922090f
     type: playbook
     task:
-      id: eaebbf2e-aa04-4581-8157-2f55adc08880
+      id: 96353c64-dc04-4475-89a8-f0f90922090f
       version: -1
-      name: Get Original Message - Gmail
-      description: ""
-      playbookName: Get Original Email - Gmail
+      name: Get Original Email - EWS_copy
```
Comment (Itay4) [inline Playbooks/playbook-Get_Original_Email_-_Generic.yml]: Please fix playbook name

File: Playbooks/playbook-Get_Original_Email_-_Generic.yml [hunk 2]
```diff
@@ -76,32 +76,52 @@ tasks:
...
+      description: |-
+        Use this playbook to retrieve the original email in the thread, including headers and attahcments, when the reporting user forwarded the original email not as an attachment.
+
+        You must have the necessary permissions in the EWS integration to execute global search: eDiscovery
+      playbookName: Get Original Email - EWS_copy
```
Comment (Itay4) [inline Playbooks/playbook-Get_Original_Email_-_Generic.yml]: Please fix playbook name

**Comment by glicht** [comment]
Please notice that the tests didn't complete. From circle:

TEST RESULTS:
	 Number of playbooks tested - 0
	 Number of succeeded tests - 0
	 Number of skipped integration - 14:
	 - Signal Sciences WAF - reason: API problems, not returning the correct data
	 - XFE - reason: Need to check the reason for skipping
	 - Symantec MSS - reason: No instance
	 - iDefense - reason: DevOps investigation
	 - Farsight DNSDB - reason: No instance
	 - VMware - reason: DevOps investigation
	 - McAfee Advanced Threat Defense - reason: No Credentials
	 - Remedy AR - reason: DevOps investigation
	 - Service Manager - reason: Expired license
	 - McAfee Active Response - reason: DevOps investigation
	 - Anomali ThreatStream - reason: Instance is down
	 - McAfee Threat Intelligence Exchange - reason: DevOps investigation
	 - Cylance Protect - reason: Under development
	 - LogRhythm - reason: DevOps investigation
	 **Number of skipped tests - 2:
	 - Phishing test - attachment - reason: can't find entryID in ExtractIndicatorsFromTextFile.
	 - Phishing test - Inline - reason: Ip range problem in IsIPInRanges**

**Comment by idovandijk** [comment]
@glicht This is interesting, as I've removed them from the "skipped tests" section in conf.json.
Any idea why they were skipped regardless?

**Pull request merged**